service/builder.go

```diff
@@ -0,0 +1,267 @@
+/*
+Copyright 2017 Caicloud Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package service
```

Comment:
After going through the refactor, I think `service` can be problematic since we are saying the framework is to build golang service ......

Reply:
how about something like `servo`?

Reply:
We can say the framework is to build `http service`.